src/styles/blocks/contact-us.scss

```diff
@@ -0,0 +1,102 @@
+.contact-us {
+  padding: 72px 20px;
+  background-color: $bg-color;
+
+  &__title {
+    font-weight: 600;
+    font-size: 32px;
+    line-height: 130%;
+    color: #fff;
+    margin-bottom: 48px;
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    box-sizing: border-box;
```

Comment:
It is a good tone to set this property for all elements(*), and don't change it without extra need